doc/classes/String.xml

```diff
@@ -385,7 +385,10 @@
 			<return type="int">
 			</return>
 			<description>
-				Converts a string containing a hexadecimal number into an integer.
+				Converts a string containing a hexadecimal number into an integer. Hexadecimal strings is expected to be pre-fixed with "[code]0x[/code]" otherwise [code]0[/code] is returned.
```

Comment:
It should be "Hexadecimal strings **are**" or "An hexadecimal string **is**".

Reply:
Fixed :slightly_smiling_face: